src/stores/RegionStore.js

```diff
@@ -351,6 +350,7 @@ export default types.model("RegionStore", {
           key,
           isArea: false,
           children: [],
+          isNotLabel: true,
```

Comment:
what's this? is it always true? or is it only for "group by label" mode? so I'd suggest to rename it